robots/gantry.h

```diff
@@ -0,0 +1,243 @@
+#pragma once
+
+#include "os/windows_include.h"
+
+// Standard C++ includes
+#include <algorithm>
+#include <chrono>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <thread>
+
+// Gantry-specifc includes
+#include "C:\Program Files\Advantech\Motion\PCI-1240\Examples\Include\Ads1240.h"
```

Comment:
p.s. I googled this to see what the hell you're driving and it is described as being "designed for general-purpose extreme motion applications" - that is awesome